src/app/home/testimonials/testimonials.component.scss

```diff
@@ -0,0 +1,28 @@
+.student-picture {
+  text-align: center;
+  min-height: 40vh;
+  display: flex;
+  align-items: center;
+
+  @media screen and (max-width: 993px) {
```

Comment:
@GigiEillen can you change this to follow what we have here and use variables from materialize?   https://github.com/gp4tech/website/blob/cf2ea4e323495fd35d18bb4c67bf245e3466e0b4/src/app/layout/footer/footer.component.scss#L10